src/main/modules/helmet/modules/contentSecurityPolicy.ts

```diff
@@ -11,8 +11,8 @@ export class ContentSecurityPolicy {
   enableFor (app: express.Express) {
     const inlineJsEnabledBodyClassName = '\'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=\''
     const inlineJsWindowGOVUKClassName = '\'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=\''
-    const scriptSrc = [inlineJsEnabledBodyClassName, inlineJsWindowGOVUKClassName, self, '*.google-analytics.com']
-    const connectSrc = [self]
+    const scriptSrc = [inlineJsEnabledBodyClassName, inlineJsWindowGOVUKClassName, self, '*.google-analytics.com', '\'unsafe-eval\'']
+    const connectSrc = [self, '*.gov.uk']
```

Comment:
Need guidance on the appropriate strategy here

Reply:
create story to expose a list of all of the non-working days, and then pull that from there instead of calling gov.uk

Reply:
ROC-5447